src/pages/integrations/angular.tsx

```diff
@@ -0,0 +1,189 @@
+          <p>
+            Tolgee for Angular is not just an i18n library. It provides much
+            more. With in-context localization feature, you can just Alt + Click
+            translations in your app and translate them directly. No more
+            extracting, no more editing of .jsons or .xliff.
+          </p>
+        </LandingPagePreviewFeature>
+        <LandingPagePreviewFeature
+          videoSrc="/landing/platform.mov"
+          bigger={true}
+        >
+          <h2>Localization platform ☁️</h2>
+          <p>
+            Tolgee localization platform provides simple user interface where,
+            you can edit all your localization string. So you don't have to
+            worry about choosing other tool for editing your translations.
+            Tolgee and it's SDKs is all you need to localize your app.
+          </p>
+        </LandingPagePreviewFeature>
+      </LandingPagePreviewFeatures>
+      <LandingPageFeatures>
+        <LandingPageFeature>
+          <h2>Simple integration 🎉</h2>
+          <p>
+            To integrate Tolgee to Angular project, you can just simply follow
+            these docs, or use integration guides in Tolgee platform. You can do
+            that in less than one minute.
+          </p>
+        </LandingPageFeature>
+        <LandingPageFeature>
+          <h2>Automated screenshot generation 📸</h2>
+          <p>
+            With in-context localization UI, you can take screenshots of your
+            application and uploaded them to Tolgee platform just by single
+            click.
+          </p>
+        </LandingPageFeature>
+
+        <LandingPageFeature>
+          <h2>Open-source 📖</h2>
+          <p>
+            Tolgee platform and integration libraries are open-source. So you
+            can self-host it and be fine!
+          </p>
+        </LandingPageFeature>
+      </LandingPageFeatures>
+      <LandingPageInstallation>
+        <LandingPageInstallationStep title="1. Create project in Tolgee platform">
+          <p>
+            Go to <a href="https://app.tolgee.io">Tolgee Cloud app</a> or access
+            your{" "}
+            <a href="/docs/server_and_web_app/self_hosting/running_with_docker">
+              self-hosted instance
+            </a>{" "}
+            and create a project. Then obtain your API key.
+          </p>
+        </LandingPageInstallationStep>
+        <LandingPageInstallationStep title="2. Setup Tolgee integration">
+          <p>Install the npm packages.</p>
+          <LandingPageInstallationCode>
+            npm i @tolgee/ngx @tolgee/ui
+          </LandingPageInstallationCode>
+          <p>Then import NgxTolgeeModule in your app</p>
+          <LandingPageInstallationCode language="ts" fullWidth>
+            {providerCode}
+          </LandingPageInstallationCode>
+        </LandingPageInstallationStep>
+        <LandingPageInstallationStep title="3. Translate your texts with pipe">
```

Comment:
Pipe is not recommended, since it's text-wrapping. Directive is span-wrapping (the element with `t` atrrubute)